app/server/src/main/scala/org/bitcoins/server/NodeRoutes.scala

```diff
@@ -22,7 +23,14 @@ case class NodeRoutes(node: Node)(implicit system: ActorSystem)
         val nodeStopping = node.stop().map { _ =>
           Server.httpSuccess("Node shutting down")
         }
-        system.terminate()
+
+        val shutdownRunnable = new Runnable {
```

Comment:
Why is this needed? Otherwise we terminate the system before propagating the http success?

Reply:
Yeah, we would queue up the message to send, but then shutdown. This was we give ourselves enough time to send the message